### PR TITLE
Update version of rocks

### PR DESCRIPTION
**Pull request by foxzi**
Update version of rocks in main(alpine), centos and ubuntu docker files
    
     - vshard to 0.1.20 version
     - queue to 1.2.0 version
     - http to 1.3.0 version
     - metrics to 0.14.0 version
     - gis to 1.0.1 version
     - mysql to 2.1.1 version

### CODE REVIEW

File: dockerfiles/centos_7 [hunk 1]
```diff
@@ -7,17 +7,18 @@ ARG TNT_VER \
...
-    LUAROCK_HTTP_VERSION=1.2.0 \
+    LUAROCK_HTTP_VERSION=1.3.0 \
     LUAROCK_MEMCACHED_VERSION=1.0.1 \
-    LUAROCK_METRICS_VERSION=0.12.0 \
+    LUAROCK_METRICS_VERSION=0.14.0 \
     LUAROCK_TARANTOOL_PG_VERSION=2.0.2 \
-    LUAROCK_TARANTOOL_MYSQL_VERSION=2.1.0 \
-    LUAROCK_TARANTOOL_GIS_VERSION=1.0.0 \
+    LUAROCK_TARANTOOL_MYSQL_VERSION=2.1.1 \
+    LUAROCK_TARANTOOL_MQTT_VERSION=1.5.1 \
```
Comment (ylobankov) [inline dockerfiles/centos_7:20]: As far as I know we shouldn't add mqtt to CentOS. https://github.com/tarantool/docker/pull/58

Comment (foxzi) [reply dockerfiles/centos_7]: done

**Review by NickVolynkin — APPROVED** [review]

**Review by ylobankov — COMMENTED** [review]
You update http to 1.3.0 version, but in the commit message you specify 1.2.0.

**Review by ylobankov — APPROVED** [review]
LGTM

**Review by NickVolynkin — APPROVED** [review]

**Pull request merged**